requirements/edx-sandbox/py38.txt

```diff
@@ -55,7 +55,7 @@ numpy==1.16.6
     #   scipy
 openedx-calc==2.0.1
     # via -r requirements/edx-sandbox/py38.in
-pillow==8.4.0
+pillow==9.0.0
```

Comment:
There isn't any breaking change mentioned in the `pillow==9.0.0` [CHANGELOG]( https://github.com/python-pillow/Pillow/blob/main/CHANGES.rst#900-2022-01-02) so it should be good to merge.